csrankings-j.csv

```diff
@@ -2519,6 +2522,7 @@ Justus Thies,TU Darmstadt,https://justusthies.github.io,4vpQvuwAAAAJ
 Juw Won Park,University of Louisville,http://louisville.edu/research/kbrin/news/juw-won-park,pTOp1TQAAAAJ
 Juyang Weng,Michigan State University,http://www.cse.msu.edu/~weng,HnQ2gqMAAAAJ
 Juyeon Jo,University of Nevada Las Vegas,http://joj5.faculty.unlv.edu,hqccsn8AAAAJ
+Jy-yong Sohn,Yonsei University,https://itml.yonsei.ac.kr/professor,Cs75s1MAAAAJ

```

Comment:
Applied stats, not CS. Please remove.

Reply:
Thank you for verifying it. I removed Jy-yong Sohn from csrankings-j.csv